app/views/home/index.html.erb

```diff
@@ -1 +1,17 @@
-<h>Hello World</h>
+<main class="home">
+  <header class="home__header"><a href="#" class="home__logo">ground rules</a></header>
+  <section class="home__section">
+    <%= render 'svg/home-ill-cup' %>
+      <h1>Created for tired people, by slightly less tired people.</h1>
+      <nav class="home__nav">
+        <ul>
+          <li><a href="">Find Your Roast</a></li>
+          <li><a href="">Master Your Machine</a></li>
+        </ul>
+      </nav>
+      <div class="hm__bodytext">
```

Comment:
Oops. I don't know why I did that in one spot. Having the - makes it a modifier I believe. Thoughts @mzlock ?

Also, would this amount of content suffice for a section? Mozilla docs is making me think otherwise since it is tied still to the heading above.